src/WebApp/Pages/Account/Forgot-Password.cshtml.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+
+using Codidact.Authentication.Domain.Entities;
+using Codidact.Authentication.Application.Common.Interfaces;
+
+namespace Codidact.Authentication.WebApp.Pages.Account
+{
+    [BindProperties]
+    public class ForgotPasswordModel : PageModel
+    {
+        private readonly UserManager<ApplicationUser> _userManager;
+        private readonly IMailService _emailService;
+
+        public ForgotPasswordModel(
+                          UserManager<ApplicationUser> userManager,
+                          IMailService emailService
+            )
+        {
+            _userManager = userManager;
+            _emailService = emailService;
+        }
+        [Required]
+        public string ReturnUrl { get; set; } = "/index";
+        public void OnGet([FromQuery] string returnUrl)
+        {
+            if (returnUrl != null)
+            {
+                ReturnUrl = returnUrl;
+            }
+        }
+
+
+        [Required(ErrorMessage = "E-Mail Address is required")]
+        [DataType(DataType.EmailAddress)]
+        public string Email { get; set; }
+
+        public bool Sent { get; set; }
+
+
+        public async Task<IActionResult> OnPostAsync()
+        {
+            if (ModelState.IsValid)
+            {
+                var user = await _userManager.FindByEmailAsync(Email);
+                if (user != null)
+                {
+                    var token = await _userManager.GeneratePasswordResetTokenAsync(user);
+                    await _emailService.SendResetPassword(user, token, ReturnUrl);
+                    Sent = true;
```

Comment:
What does `Sent` do?

Reply:
Inside Forgot-password.cshtml https://github.com/codidact/authentication/pull/19/files/3756f5d68bd71b3de0a9312951dae7411768f2c4#diff-4116e1d5bcb44102bbbff076c88b9c70R11 it tells if wheater the email has been sent or not

Reply:
I overlooked that, sorry.